.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+  - "2.6"
+  - "2.7"
+  - "3.2"
+  - "3.3"
+  - "3.4"
+  - "3.5"
+
+# Test
+script: nosetest -v
```

Comment:
you should put line break at the and of file

http://stackoverflow.com/questions/5813311/no-newline-at-end-of-file